crates/store/src/db/migrations.rs

```diff
@@ -100,8 +100,8 @@ fn compute_migration_hashes() -> Vec<Hash> {
 }
 
 fn preprocess_sql(sql: &str) -> String {
-    // TODO: We can also remove all comments here (need to analyze the SQL script in order to remain comments
-    //       in string literals).
+    // TODO: We can also remove all comments here (need to analyze the SQL script in order to remain
+    // comments       in string literals).
```

Comment:
nit: some extra spaces on the second line.

Reply:
Pity one can't customise rustfmt with additional rules like aligning indents for todo's etc. Thanks!